src/page/items/AllProductList.js

```diff
@@ -1,5 +1,6 @@
 import { useState } from "react";
 import ProductItem from "./ProductItem";
+import { Navigate } from "react-router-dom";
 
 function AllProductListItem({ product, className }) {
```

Comment:
```suggestion
function AllProductListItem({ product }) {
```
여기 className은 사용이 안되고있는것같아요!
사용하지 않는 props는 선언시 제외해주시면 코드가 조금 더 깔끔해 집니다!